command/commands/token.py

```diff
@@ -0,0 +1,77 @@
+"""Command to obtain signed authentication token."""
+import jwt
+import logging
+
+from datetime import datetime
+from model.permissions import Permissions
+
+
+class TokenCommand:
+    """Token command model class."""
```

Comment:
I noticed that we don't initialize parsers here like in the other commands which confuses me ? Or is this something that is to be added later

Reply:
We don't need parsers, there's no subcommands/options.